pkg/agent/cniserver/server_windows.go

```diff
@@ -98,3 +99,13 @@ func getInfraContainer(containerID, netNS string) string {
 func (c *CNIConfig) getInfraContainer() string {
 	return getInfraContainer(c.ContainerId, c.Netns)
 }
+
+// getPodsListOptions returns the Pods running on the current Node. Note, the host-network Pods are not filtered
+// out on Windows because they are also managed by antrea as long as "spec.SecurityContext.windowsOptions.hostProcess"
```

Comment:
what will happen if "spec.SecurityContext.windowsOptions.hostProcess" is configured and the Pod is not filtered out?